great_expectations/cli/datasource.py

```diff
@@ -84,51 +114,83 @@ def _add_pandas_datasource(context):
     return data_source_name
 
 
-def _add_sqlalchemy_datasource(context):
+def load_library(library_name, install_instructions_string=None):
```

Comment:
This is a great convenience, but I think it would be much nicer to make library name an iterable so that we can check for drivers at the same time instead of having people fail with a single error at a time until it works.

Reply:
Created a task for 0.8.5